### ISSUE TITLE
v3 does not use package-name from config

### ISSUE THREAD

**Issue by atreya2011** [google-github-actions/release-please-action#413]
### TL;DR

v2 of the action respects `package-name` in the configuration. However, v3 of the action just uses the latest merge commit message as the release title. Is there any way to override this? Please let me know if I have missed some config in the docs because I am unable to find a way to override the release title in v3.

Screenshot for your reference. v1.3.1 is created using v2 whereas v1.3.0 is created using v3
<img width="1042" alt="Screen Shot 2022-01-06 at 22 32 57" src="https://user-images.githubusercontent.com/44151328/148390711-8efc811a-2411-41d7-a2b7-2d9c4d6533d3.png">

### Expected behavior

v3 of the action should use the `package-name` from config for the release title

### Observed behavior

v3 uses merge commit message instead for the release title

### Action YAML

```yaml
name: Release

on:
  push:
    branches:
      - main

jobs:
  release:
    runs-on: ubuntu-latest
    steps:
      - name: GitHub Release
        uses: GoogleCloudPlatform/release-please-action@v2
        id: release
        continue-on-error: true
        with:
          token: ${{ github.token }}
          release-type: node
          package-name: "grpc-proto-laughing-brocolli"
          command: github-release

      - name: Checkout code
        uses: actions/checkout@v2
        if: ${{ steps.release.outputs.release_created }}

      - name: Setup Node
        uses: actions/setup-node@v2
        if: ${{ steps.release.outputs.release_created }}
        with:
          node-version: 14
          registry-url: 'https://npm.pkg.github.com/'
          scope: '@atreya2011'

      - name: Publish
        id: publish
        if: ${{ steps.release.outputs.release_created }}
        env:
          NODE_AUTH_TOKEN: ${{ github.token }}
        run: |
          make npm-install
          make publish-npmpkg
```

### Additional information
I am not sure if this is a similar issue though... 
https://github.com/google-github-actions/release-please-action/issues/260

**Comment by bcoe** [comment]
@atreya2011 thank you for the bug report.

**Comment by chingor13** [comment]
I believe this was fixed very early on in v13 of release-please - it was incorrectly grabbing release notes from the pull request to use as the release body and thus the release title/body shows the commit message of the release SHA.
